integration-cli/docker_cli_swarm_test.go

```diff
@@ -837,6 +837,45 @@ func (s *DockerSwarmSuite) TestSwarmServiceTTYUpdate(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Equals, "true")
 }
 
+func (s *DockerSwarmSuite) TestSwarmServiceNetworkUpdate(c *check.C) {
+	d := s.AddDaemon(c, true, true)
+
+	out, err := d.Cmd("network", "create", "-d", "overlay", "foo")
```

Comment:
`d.Cmd()` needs to be deprecated, please use `d.Command()` which provides better errors messages and less verbose test cases.